beard/clustering/blocking.py

```diff
@@ -37,6 +43,54 @@ def predict(self, X):
         return _single(X)
 
 
+def _parallel_fit(X, y, blocks, affinity, fit_, partial_fit_, base_estimator,
+                  clusterers_, bs):
+
+    res = []
```

Comment:
Can you rename this to `clusterers` and make it a dictionary? This way we would mimic the interface of `clusterers_`.